types/react-dropzone/react-dropzone-tests.tsx

```diff
@@ -1,44 +1,49 @@
-import * as React from 'react';
-import Dropzone = require('react-dropzone');
+import * as React from "react";
+import * as Dropzone from "react-dropzone";
 
 class Test extends React.Component {
-  constructor(props: any) {
-    super(props);
-  }
 
   dz: Dropzone;
 
   open() {
     this.dz.open();
   }
 
+  handleDropFile = (files: File[]) => {};
+
+  handleDropFiles = (accepted: File[], rejected?: File[]) => {};
+
   render() {
     return (
       <div>
         <Dropzone
           ref={(node) => { this.dz = node } }
-          onDrop={(e: any) => { e.preventDefault(); } }
-          onDropAccepted={(e: any) => { e.preventDefault(); } }
-          onDropRejected={(e: any) => { e.preventDefault(); } }
-          onDragStart={(e: any) => { e.preventDefault(); } }
-          onDragEnter={(e: any) => { e.preventDefault(); } }
-          onDragLeave={(e: any) => { e.preventDefault(); } }
+          onClick={ (e: any) => { e.preventDefault(); } }
+          onDrop={ this.handleDropFiles }
+          onDropAccepted={ this.handleDropFile }
+          onDropRejected={ this.handleDropFile }
+          onDragStart={ (e: any) => { e.preventDefault(); } }
+          onDragEnter={ (e: any) => { e.preventDefault(); } }
+          onDragLeave={ (e: any) => { e.preventDefault(); } }
+          onFileDialogCancel={ () => undefined }
           style={{ borderStyle: "dashed" }}
           activeStyle={{ borderStyle: "dotted" }}
           rejectStyle={{ borderStyle: "dotted" }}
           className="regular"
           activeClassName="active"
           rejectClassName="reject"
-          minSize={2000}
-          maxSize={Infinity}
-          disablePreview={true}
-          disableClick={true}
-          multiple={false}
+          minSize={ 2000 }
```

Comment:
keep the spacing around values consistent ( the v2 typings do not have spaces ). i'd keep the original